clayui.com/src/styles/_404-page.scss

```diff
@@ -0,0 +1,66 @@
+.clay-404-page-container {
+	background-image: url('../../static/images/home/bg_white.svg');
+
+	header {
+		background-color: #fff;
+
+		img {
+			width: 32px;
+		}
+	}
+
+	.navbar {
+		width: 100%;
+	}
+
+	.list-inline {
+		a {
+			background-color: $brand-primary;
+			color: #fff;
+			transition: all 0.3s ease-in-out;
+
+			.githubImage {
+				width: 16px;
+				height: 16px;
+				background-size: cover;
+				background-image: url('../../static/images/home/GitHub-Mark-64px.svg');
+			}
+
+			&:hover {
+				background-color: $brand-dark;
+				color: $brand-light;
+				box-shadow: 0 0 0 !important;
+
+				svg {
+					color: $brand-light;
+				}
```

Comment:
Remove this, `lexicon-icon` class inherits color so `$brand-light` will cascade to the svgs.